lib/logger.js

```diff
@@ -44,31 +44,49 @@ const isValidTimestamp = (timestamp) => {
 
 function Logger(key, options) {
     if (!(this instanceof Logger)) return new Logger(key, options);
-    checkParam(key, 'LogDNA Ingestion Key', false);
-    checkParam(options.hostname, 'Hostname', true);
-    checkParam(options.mac, 'MAC Address', true);
-    checkParam(options.ip, 'IP Address', true);
-    checkParam(options.level, 'Level', true);
-    checkParam(options.app, 'App', true);
-    checkParam(options.tags, 'Tags', true);
-    checkParam(options.logdna_url, 'LogDNA URL', true);
-    if (options.timeout) {
-        if (!isInt(options.timeout)) {
-            throw new Error('Timeout must be an Integer');
+
+    checkStringParam(key, 'LogDNA Ingestion Key', false);
+    checkStringParam(options.hostname, 'Hostname', true);
+    checkStringParam(options.mac, 'MAC Address', true);
+    checkStringParam(options.ip, 'IP Address', true);
+    checkStringParam(options.level, 'Level', true);
+    checkStringParam(options.app, 'App', true);
+    checkStringParam(options.tags, 'Tags', true);
```

Comment:
This will still throw an error if options.tags is an array and not a string

Reply:
Forgot to remove the line - now it's done!